Text, Heading: Ensure `maxLines` truncates correctly in flex container (#1580)
seek-oss/braid-design-system: .changeset/hungry-ads-cheer.md

---
'braid-design-system': patch
---

---
updated:
  - Text
  - Heading
---

**Text, Heading:** Ensure `maxLines` truncates correctly in flex container

> Provide `min-width` to `Text` and `Heading` components to ensure `maxLines` truncates as expected when typography components are used as a direct child of a flex container.

Reviewer: Feel like this is implementation detail. Could honestly confuse users into thinking they need to make a change.

> **Text, Heading:** Ensure `maxLines` truncates correctly in flex container
> 
> Provide `min-width` to `Text` and `Heading` components to ensure `maxLines` truncates as expected when typography components are used as a direct child of a flex container.

Author: ```suggestion
**Text, Heading:** Ensure `maxLines` truncates correctly when used as the direct child of a flex container.
```
